packages/rocketchat-switch/rocketchat-switch.html

```diff
@@ -0,0 +1,21 @@
+<template name="switch">
```

Comment:
If you are creating a new component please replace the places where it is used to use the component